plugins/cpp/test/src/cpptest.cpp

```diff
@@ -0,0 +1,27 @@
+#define GTEST_HAS_TR1_TUPLE 1
```

Comment:
Changes in this file should be moved in your other commit (`Adding C++ parser test`).

Reply:
I melded the two commits into one. The first one made no sense.